packages/contracts-bedrock/contracts/L2/L2ToL1MessagePasser.sol

```diff
@@ -107,9 +113,29 @@ contract L2ToL1MessagePasser is Semver {
 
         sentMessages[withdrawalHash] = true;
 
-        emit MessagePassed(nonce, msg.sender, _target, msg.value, _gasLimit, _data, withdrawalHash);
+        emit MessagePassed(
+            messageNonce(),
```

Comment:
The migrated withdrawals will be a special case, where they do not have the version applied to the nonce, but they are hashing with version 1 style hashing

Reply:
That's fine, we just say that version 0 and 1 have the same hashing scheme